src/Learning/KWLearningProblem/CMakeLists.txt

```diff
@@ -18,3 +20,23 @@ target_link_libraries(
          KWUserInterface
          SNBPredictor
          MHHistograms)
+
+# cmake-format: off
```

Comment:
why cmake-format is off here ?

Reply:
Because cmake-format produces an unreadable thing, difficult to maintain:
```cmake
generate_gui(
  KWModelingAdvancedSpec
  "Advanced predictor parameters"
  KWModelingAdvancedSpec.dd
  KWModelingAdvancedSpec.dd.log
  -nomodel
  -noarrayview
  -specificmodel
  KWModelingSpec)
generate_gui(
  KWModelingExpertSpec
  "Expert predictor parameters"
  KWModelingExpertSpec.dd
  KWModelingExpertSpec.dd.log
  -nomodel
  -noarrayview
  -specificmodel
  KWModelingSpec)
```

Reply:
Ok